modules/ducktests/tests/ignitetest/services/ignite_app.py

```diff
@@ -38,33 +41,54 @@ def __init__(self, context, config, java_class_name, params="", timeout_sec=60,
         self.servicejava_class_name = servicejava_class_name
         self.java_class_name = java_class_name
         self.timeout_sec = timeout_sec
-        self.stop_timeout_sec = 10
+        self.params = params
```

Comment:
How is it used?

Reply:
It is used to gain start params of the application.
see cellular_affinity_test.py:155 for example.

Reply:
ok